src/WidgetRegistry.ts

```diff
@@ -78,7 +86,7 @@ export class WidgetRegistry extends Evented implements WidgetRegistry {
 			item.then((widgetCtor) => {
 				this.registry.set(widgetLabel, widgetCtor);
 				this.emit({
-					type: `loaded:${widgetLabel}`
+					type: getEventType(widgetLabel)
```

Comment:
As above, how would I register for an event that has a Symbol label and how would that be different than other possible events?  Usually `type` is some sort of "class" in an event and there is other stuff in the event that talks about the specifics.  Shouldn't it really be `target`?